UserDetails/migrations/0013_merge_20190429_1911.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.1.5 on 2019-04-29 17:11
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('UserDetails', '0012_auto_20190429_1306'),
+        ('UserDetails', '0011_auto_20190429_1748'),
+    ]
+
+    operations = [
+    ]
```

Comment:
These 2 migrations are probably going to be problematic because migrations up to 12 are already applied on the server.  Better to take the migrations from `master` and then create a new migration.

Reply:
I made a migration file before I had a local copy of the migration files from the Wessels branch. The last migration file is a merge file, so it should not have a conflict. But I changed it anyhow.